recipes/qt5/meta.yaml

```diff
@@ -41,12 +52,12 @@ requirements:
     - curl  # [win]
     - flex  # [linux]
     - fontconfig 2.11.*  # [linux]
-    - freetype  2.5* # [linux]
+    - freetype  2.5*  # [linux]
     - gst-plugins-base  # [linux]
-    - icu 56.*
```

Comment:
I think we need to keep ICU pinned because its libraries have the SO version in them (icu56.dll)